packages/belos/src/BelosCGSingleRedIter.hpp

```diff
@@ -398,6 +427,22 @@ class CGSingleRedIter : virtual public CGIteration<ScalarType,MV,OP> {
   }
 
 
+  //////////////////////////////////////////////////////////////////////////////////////////////////
+  // Get the norms of the residuals native to the solver.
+  template <class ScalarType, class MV, class OP>
+  Teuchos::RCP<const MV>
+  CGSingleRedIter<ScalarType,MV,OP>::getNativeResiduals( std::vector<MagnitudeType> *norms ) const {
```

Comment:
It's technically possible to call this method with `norms == nullptr`, though I suppose the "iter" classes are just implementation details of the "SolMgr" classes.